webview-sample/README.md

```diff
@@ -32,3 +32,16 @@ Demonstrates VS Code's [webview API](https://code.visualstudio.com/api/extension
 - `F5` to start debugging
 
 Run the `Cat Coding: Start cat coding session` to create the webview.
+
+## Commands
+
+This extension provides the following commands:
+
+- `Cat Coding: Start cat coding session`: Creates and displays the Cat Coding webview.
```

Comment:
nit: these commands are usually using title casing—could you just double check that this is exactly how they show up in the command palette?